test/infrastructure/kit-operator.cloudformation.yaml

```diff
@@ -0,0 +1,75 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Resources used by https://github.com/awslabs/kubernetes-iteration-toolkit/operator
+Parameters:
+  ClusterName:
+    Type: String
+    Description: "Substrate/host cluster name"
+Resources:
+  KitControllerPolicy:
```

Comment:
I had it in so that we could make changes to KIT as desired quickly, but as we're going to depend on KIT for this in the future, I don't think it's worth keeping it here.